internal/session/query.go

```diff
@@ -334,9 +334,9 @@ with
       returning public_id, server_id
   )
   select
-    dead_servers.private_id,
-    dead_servers.update_time,
-    count(closed_connections.public_id)
+    dead_servers.private_id as server_id,
```

Comment:
These got named via `as` but those names aren't actually then used in the statement below.

Reply:
@jefferai this was due to fbccee6df93743d1a77c03d1da32b3f6c53e57f3. Unless there's a better way to make these results available in a struct thru gorm?

This was done after https://github.com/hashicorp/boundary/pull/1400/commits/1ee29688da118da6d622b72024701845de6b5a44.

Reply:
In turn related to https://github.com/hashicorp/boundary/pull/1340#discussion_r669944414

Reply:
There's no issue with having names assigned...I'm simply calling out that since they are being given a name, other parts of the query could use the name to simplify things.